src/TokenManager.ts

```diff
@@ -15,6 +15,8 @@
 
 import * as jwt from 'jsonwebtoken';
 import {Token} from './Token';
+import {IVerifyOptions} from './IVerifyOptions';

```

Comment:
I would rename `IVerifyOptions` to `IJWTVerifyOptions` to give some context on the name.